### PR TITLE
[LAKESIDE-166] Change the default system responses

### PR DESCRIPTION
**Pull request by dyudyunov**
while submitting answers in Python-Evaluated Input Block

**Youtrack:** https://youtrack.raccoongang.com/issue/FLS-166

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: common/lib/xmodule/xmodule/capa_base.py [hunk 1]
```diff
@@ -792,32 +792,46 @@ def _get_answer_notification(self, render_notifications):
                         break
 
             # Build the notification message based on the notification type and translate it.
+            data = getattr(self, 'data', '')
+            # Customization for the default system responses while submitting answers in Python-Evaluated Input Block
+            # by `python_evalueted_block` flag. The `data.find('<jsinput') < 0` check provided to exclude Custom
```
Comment (cmltaWt0) [inline common/lib/xmodule/xmodule/capa_base.py]: Typo: `python_evalueted_bloc` -> `python_evaluated_bloc`

File: common/lib/xmodule/xmodule/capa_base.py [hunk 2]
```diff
@@ -792,32 +792,46 @@ def _get_answer_notification(self, render_notifications):
...
+            # JavaScript Display and Grading block.
+            python_evalueted_block = data.find('<script type="loncapa/python">') >= 0 and data.find('<jsinput') < 0
```
Comment (cmltaWt0) [inline common/lib/xmodule/xmodule/capa_base.py]: Typo?

File: common/lib/xmodule/xmodule/capa_base.py [hunk 3]
```diff
@@ -792,32 +792,46 @@ def _get_answer_notification(self, render_notifications):
...
-                        progress.frac()[1]
-                    ).format(progress=str(progress))
+                    if not python_evalueted_block:
+                        answer_notification_message = ungettext(
+                            "Incorrect ({progress} point)",
+                            "Incorrect ({progress} points)",
+                            progress.frac()[1]
+                        ).format(progress=str(progress))
+                    else:
+                        answer_notification_message = _('Answer submission required')
```
Comment (cmltaWt0) [inline common/lib/xmodule/xmodule/capa_base.py]: Have you added any translations for this answer?

File: common/lib/xmodule/xmodule/capa_base.py [hunk 4]
```diff
@@ -792,32 +792,46 @@ def _get_answer_notification(self, render_notifications):
...
-                        progress.frac()[1]
-                    ).format(progress=str(progress))
+                    if not python_evalueted_block:
+                        answer_notification_message = ungettext(
+                            "Correct ({progress} point)",
+                            "Correct ({progress} points)",
+                            progress.frac()[1]
+                        ).format(progress=str(progress))
+                    else:
+                        answer_notification_message = _('Answer submitted')
```
Comment (cmltaWt0) [inline common/lib/xmodule/xmodule/capa_base.py]: The same as the above.

File: common/lib/xmodule/xmodule/capa_base.py (hunk 3 above)
Comment (cmltaWt0) [inline common/lib/xmodule/xmodule/capa_base.py]: Also, why don't you use `ungettext`?

File: common/lib/xmodule/xmodule/capa_base.py [hunk 5]
```diff
@@ -792,32 +792,46 @@ def _get_answer_notification(self, render_notifications):
...
-                        progress.frac()[1]
-                    ).format(progress=str(progress))
+                    if not python_evalueted_block:
+                        answer_notification_message = ungettext(
+                            "Partially correct ({progress} point)",
+                            "Partially correct ({progress} points)",
+                            progress.frac()[1]
+                        ).format(progress=str(progress))
+                    else:
+                        answer_notification_message = _('Answer submitted')
```
Comment (cmltaWt0) [inline common/lib/xmodule/xmodule/capa_base.py]: The same as the above.

File: common/lib/xmodule/xmodule/capa_base.py [hunk 6]
```diff
@@ -792,32 +792,46 @@ def _get_answer_notification(self, render_notifications):
...
                 else:
                     answer_notification_message = _('Correct')
             elif answer_notification_type == 'partially-correct':
                 if progress is not None:
-                    answer_notification_message = ungettext(
-                        "Partially correct ({progress} point)",
-                        "Partially correct ({progress} points)",
-                        progress.frac()[1]
-                    ).format(progress=str(progress))
+                    if not python_evalueted_block:
```
Comment (cmltaWt0) [inline common/lib/xmodule/xmodule/capa_base.py]: Typo?

File: common/lib/xmodule/xmodule/capa_base.py [hunk 7]
```diff
@@ -792,32 +792,46 @@ def _get_answer_notification(self, render_notifications):
...
                 else:
                     answer_notification_message = _('Incorrect')
             elif answer_notification_type == 'correct':
                 if progress is not None:
-                    answer_notification_message = ungettext(
-                        "Correct ({progress} point)",
-                        "Correct ({progress} points)",
-                        progress.frac()[1]
-                    ).format(progress=str(progress))
+                    if not python_evalueted_block:
```
Comment (cmltaWt0) [inline common/lib/xmodule/xmodule/capa_base.py]: Typo?

File: common/lib/xmodule/xmodule/capa_base.py (hunk 3 above)
Comment (dyudyunov) [reply common/lib/xmodule/xmodule/capa_base.py]: Messages applied according to the ticket. There are no plural forms for messages.

Comment (cmltaWt0) [reply common/lib/xmodule/xmodule/capa_base.py]: Ok. What about translations? Do we need it?

File: common/lib/xmodule/xmodule/capa_base.py (hunk 7 above)
Comment (dyudyunov) [reply common/lib/xmodule/xmodule/capa_base.py]: Fixed

File: common/lib/xmodule/xmodule/capa_base.py (hunk 6 above)
Comment (dyudyunov) [reply common/lib/xmodule/xmodule/capa_base.py]: Fixed

File: common/lib/xmodule/xmodule/capa_base.py (hunk 2 above)
Comment (dyudyunov) [reply common/lib/xmodule/xmodule/capa_base.py]: Fixed

File: common/lib/xmodule/xmodule/capa_base.py (hunk 1 above)
Comment (dyudyunov) [reply common/lib/xmodule/xmodule/capa_base.py]: Fixed

File: common/lib/xmodule/xmodule/capa_base.py (hunk 3 above)
Comment (dyudyunov) [reply common/lib/xmodule/xmodule/capa_base.py]: No, we don't need them.

**Pull request merged**